web_app/db/crud.py

```diff
@@ -89,6 +89,27 @@ def get_object_by_field(
         finally:
             db.close()
 
+
+    def get_all_users_with_opened_position(self) -> List[User]:
+        """
+        Retrieves all users with an OPENED position status from the database.
+        
+        :return: List[User]
+        """
+        with self.Session() as db:
+            try:
+                return (
+                    db.query(User)
```

Comment:
Please, use `Position` and get from there all users. It will be definetly better way instead of having 2 joins